kairon/importer/validator/file_validator.py

```diff
@@ -178,13 +182,21 @@ def verify_intents_in_stories(self, raise_exception: bool = True):
         """
         intents_mismatched = []
         self.verify_intents(raise_exception)
+        multiflow_stories_intent = {}
 
         stories_intents = {
             event.intent["name"]
             for story in self.story_graph.story_steps
             for event in story.events
             if isinstance(event, UserUttered)
         }
+        if self.multiflow_stories:
```

Comment:
why shoudnt we create a graph out of it and use it to find intents. That would be an easier to read code.